### PR TITLE
✨ Return next new NFT ID for purchase info

### PR DESCRIPTION
**Pull request by nwingt**
For liker.land to retrieve new NFT ID of NFT Class in the new NFT page

### CODE REVIEW

**Comment by williamchong** [comment]
What is this ID for? as the comment above sugggest, NFT ID is fetched live when processing purchase transaction to prevent purchase of same price level from blocking each other
https://github.com/likecoin/likecoin-api-public/blob/91ac85cf212afd124819fd474501eb041fc4f7b3/src/util/api/likernft/purchase.js#L344

**Comment by nwingt** [comment]
In the new design, users will go to the NFT details page instead NFT Class details page, so I need to obtain the **FIRST** unsold NFT ID of that NFT Class and redirect users to the proper NFT details page

Maybe there are some situations that haven't been covered

And I guess users may not collect the same NFT ID shown on the NFT details page as someone else has probably collected that
image

**Review by williamchong — APPROVED** [review]
Ok if we don't use this ID thinking it is the purchased ID

File: src/util/api/likernft/purchase.js [hunk 1]
```diff
@@ -103,6 +104,7 @@ export async function getLatestNFTPriceAndInfo(iscnPrefix, classId) {
   if (newNftData) {
     price = currentPrice;
     // Do not set NFT ID since another fresh one might be used on purchase instead
```
Comment (williamchong) [inline src/util/api/likernft/purchase.js]: Maybe modify this comment though

**Review by williamchong — CHANGES_REQUESTED** [review]

**Pull request merged**